cleanlab/object_detection/rank.py

```diff
@@ -169,11 +175,12 @@ def issues_from_scores(label_quality_scores: np.ndarray, *, threshold: float = 0
 def _compute_label_quality_scores(
     labels: List[Dict[str, Any]],
     predictions: List[np.ndarray],
-    aggregation_weights: Optional[Dict[str, float]] = None,
-    *,
     method: str = "objectlab",
-    threshold: Optional[float] = None,
     verbose: bool = True,
```

Comment:
same w verbose arg. Please go through ALL of the object detection code and move all optional args below the * in user-facing methods